package.json

```diff
@@ -4,12 +4,14 @@
   "dependencies": {
     "alex": "~2",
     "autoprefixer": "~6",
+    "babel-eslint": "~6",
```

Comment:
they are not needed in the standard usecase but in a special one where eslint needs additional deps, we're using that in tests so it's not in here. If you feel this is wrong I'd file an issue as this commit is already in master.